### PR TITLE
feature/DPP-61

### PR DESCRIPTION
**Pull request by maikel-maas**

### CODE REVIEW

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 1]
```diff
@@ -61,6 +62,13 @@ const getCellFormatting = (type: TableCellTypeEnum, value: any, truncationLength
       const labels = Array.isArray(value) ? value.map((label: LabelTypeEnum) => <SSITypeLabel type={label} />) : <SSITypeLabel type={value} />
       return <LabelCell>{labels}</LabelCell>
     }
+    case TableCellTypeEnum.STATUS: {
+      return <SSIStatusLabel style={{
+        width: '60px',
```
Comment (BtencateSphereon) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:67]: why is there a width and a height? the status component should scale based on the content, which it should already do

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 2]
```diff
@@ -61,6 +62,13 @@ const getCellFormatting = (type: TableCellTypeEnum, value: any, truncationLength
...
+        height: '15px',
+        justifyContent: 'center'
```
Comment (BtencateSphereon) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:69]: this should already be handles by the component

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]

File: packages/core/src/styles/ssi/colors.ts [hunk 3]
```diff
@@ -30,6 +30,12 @@ export const statusColors: Record<LabelStatus | 'error', string> = {
   verified: '#00C249',
   unverified: '#FF9900',
   error: '#D74500',
+  New: '#7276F7',
+  Approved: '#00C249',
+  Pending: '#0B81FF',
+  Archived: '#B3B3B3',
+  Declined: '#D74500',
+  Done: '#000' // TODO I see it in the UI but not in the design, need to update it with the correct color
```
Comment (BtencateSphereon) [inline packages/core/src/styles/ssi/colors.ts:38]: would be nice to create a ticket about this

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 4]
```diff
@@ -27,6 +27,7 @@ import {
   SSITableViewTableContainerStyled as TableContainer,
 } from '../../../styles'
 import {Button, ColumnHeader, TableCellType} from '../../../types'
+import {SSIStatusLabel} from "../../../index";
```
Comment (BtencateSphereon) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:30]: single quotes

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 5]
```diff
@@ -61,6 +62,9 @@ const getCellFormatting = (type: TableCellType, value: any, truncationLength?: n
       const labels = Array.isArray(value) ? value.map((label: LabelType) => <SSITypeLabel type={label} />) : <SSITypeLabel type={value} />
       return <LabelCell>{labels}</LabelCell>
     }
+    case TableCellType.STATUS: {
+      return <SSIStatusLabel status={value as LabelStatus} />
```
Comment (BtencateSphereon) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:66]: should we not use the WorkFlowStatus as value for the column and then use it here?

File: packages/ssi-react/src/components/views/SSITableView/index.tsx (hunk 4 above)
Comment (maikel-maas) [reply packages/ssi-react/src/components/views/SSITableView/index.tsx:30]: Fixed

File: packages/core/src/styles/ssi/colors.ts (hunk 3 above)
Comment (maikel-maas) [reply packages/core/src/styles/ssi/colors.ts:38]: Fixed

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**